client/src/components/CheckBox/CheckBox.tsx

```diff
@@ -41,12 +36,12 @@ export class CheckBox extends React.Component<CheckBoxProps> {
       label,
       active,
       onClick,
-      color,
-      isSolidBox,
+      color = primaryColor,
+      isSolidBox = false,
       container_style,
       checkbox_style,
       label_style,
-      checkmark_vertical_align,
+      checkmark_vertical_align = 0.1,
       disabled,
     } = this.props;
```

Comment:
For a class component, `static defaultProps` is still the correct solution I think. The class could be, now or in the future, accessing `this.props` in multiple places (e.g. constructor, life cycle methods, etc). Would have to manually repeat these defaults in all `props` destructuring/direct accessing of `this.props`. Oof, also in things like `preProps` args to certain life cycle methods.

I think the advice about setting defaults inside prop destructuring is for functional components. Totally makes sense in that case.